examples/chart/teleport-cluster/values.schema.json

```diff
@@ -69,6 +69,11 @@
             "type": "string",
             "default": ""
         },
+        "sessionRecording": {
```

Comment:
What do you think about adding an enum here?

Reply:
Yeah, it's kind of an issue

All of this should be auto-generated
Reference docs and configuration knobs